helper/builtinplugins/registry.go

```diff
@@ -283,6 +283,11 @@ func (r *registry) DeprecationStatus(name string, pluginType consts.PluginType)
 	return consts.Unknown, false
 }
 
+// IsBuiltinEntPlugin checks whether the plugin is an enterprise only builtin plugin
+func (r *registry) IsBuiltinEntPlugin(name string, pluginType consts.PluginType) bool {
```

Comment:
I think you're going to have to handle both OSS and ENT here. This file is in both OSS and ENT.